### ISSUE TITLE
Docs for "OperatingSystemKind" condition?

### ISSUE THREAD

**Issue by rmunn** [dotnet/templating#1421]
In https://github.com/dotnet/templating/pull/1042, @mlorbetske gives a sample post-action with `"condition": "(OS != \"windows\")"` so that it will only run on non-Windows systems. But in https://github.com/SAFE-Stack/SAFE-template/issues/36, it seems that that condition doesn't actually work to exclude a post-action from a Windows machine, causing the template installation to fail. (That particular post-action runs the Unix `chmod` command, so it will *only* work on Linux or OS X machines). I've found [one reference to a condition called `OperatingSystemKind`](https://github.com/dotnet/templating/blob/04744d0c1e90b4b582baf5216f2f4e5663f1187d/test/Microsoft.TemplateEngine.Orchestrator.RunnableProjects.UnitTests/TemplateConfigTests/PostActionTests.cs), but I haven't found any documentation for that condition. What values is it expected to have? How should I use it to ensure that a particular post-action only runs on Unix-like systems?

**Comment by mlorbetske** [comment]
I believe the conditioning for "Windows or not" needs a value of `Windows_NT` (the value of the `OS` environment variable since Windows 2000/XP)

**Comment by rmunn** [comment]
Yep, changing the condition to `"(OS != \"Windows_NT\")" ` fixed https://github.com/SAFE-Stack/SAFE-template/issues/36. And on further research, I found that the `OperatingSystemKind` variable I was asking about was a config variable set up just for those specific unit tests, and is not something generally available. So `OS` is the environment variable to check.

And this issue can be closed: now that I know how to check for Windows properly, I've updated the [documentation for the `chmod` post-action](https://github.com/dotnet/templating/wiki/Post-Action-Registry#change-file-permissions) to have the right condition, so there's no further action needed on this AFAICT.
